Makefile

```diff
@@ -26,8 +28,15 @@ CC_VERSION=$(shell $(CC) -v 2>&1|grep -E '(gcc|clang) version')
 
 ifneq (,$(findstring gcc,$(CC_VERSION)))
 	CFLAGS+=-Wno-unused-but-set-variable			\
-		-Wno-zero-length-bounds				\
 		-Wno-stringop-overflow
+#gcc 7-9 do not allow the -Wno-zero-length-bounds, and github currently still uses gcc 7, so we'll have to exclude it
```

Comment:
Let's do this the way the kernel does it - grep for cc-disable-warning in the kernel source tree and steal that.